src/ipc-handlers.ts

```diff
@@ -676,8 +682,35 @@ export async function exportSite(
 			const parentWindow = BrowserWindow.fromWebContents( event.sender );
 			sendIpcEventToRendererWithWindow( parentWindow, 'on-export', data, siteId );
 		};
-		return await exportBackup( options, onEvent );
+
+		const result = await exportBackup( options, onEvent );
+
+		if ( result ) {
+			bumpStat( STATS_GROUP.STUDIO_EXPORT, STATS_METRIC.SUCCESS );
+
+			// Track which content types are being included in the export
+			if ( options.includes ) {
+				if ( options.includes.database ) {
+					bumpStat( STATS_GROUP.STUDIO_EXPORT_CONTENT, STATS_METRIC.DATABASE );
+				}
+				if ( options.includes.uploads ) {
+					bumpStat( STATS_GROUP.STUDIO_EXPORT_CONTENT, STATS_METRIC.UPLOADS );
+				}
+				if ( options.includes.plugins ) {
+					bumpStat( STATS_GROUP.STUDIO_EXPORT_CONTENT, STATS_METRIC.PLUGINS );
+				}
+				if ( options.includes.themes ) {
+					bumpStat( STATS_GROUP.STUDIO_EXPORT_CONTENT, STATS_METRIC.THEMES );
+				}
+				if ( options.includes.muPlugins ) {
+					bumpStat( STATS_GROUP.STUDIO_EXPORT_CONTENT, STATS_METRIC.MU_PLUGINS );
+				}
+			}
```

Comment:
I don't think we need this granularity in the export.
We have only two buttons in the UI.
One to export the whole site and another to export the database.
What if we expose the bumpStat to ipc-handlers and call it onClick on the UI?

https://github.com/Automattic/studio/blob/edf000ca48d2ff4f8a98e564fb32dc6c5c5988c2/src/components/content-tab-import-export.tsx#L77-L94